apiserver/plane/app/views/issue_type/issue_type_custom_property.py

```
@@ -0,0 +1,53 @@
+from django.db import IntegrityError
+
+from rest_framework import status
+from rest_framework.response import Response
+
+from plane.api.serializers import IssueTypeCustomPropertySerializer
+from plane.db.models import Workspace, IssueTypeCustomProperty
+from plane.app.views import BaseAPIView
+
+class IssueTypeCustomPropertyAPIEndpoint(BaseAPIView):
+    def get(self, request, slug, issue_type, pk=None):
+        workspace = Workspace.objects.get(slug=slug)
```

Ruff (0.11.9)

Local variable `workspace` is assigned to but never used

Remove assignment to unused variable `workspace`

(F841)